### PR TITLE
Upgrading to v2.6.1

### PR DESCRIPTION
**Pull request by metinavci**
SP-30 Reduce Timeout and Re-connection Duration

### CODE REVIEW

File: SPIClient/Connection.cs [hunk 1]
```diff
@@ -85,7 +85,7 @@ public void Connect()
             new Thread(() =>
             {
                 Thread.CurrentThread.IsBackground = true;
-                Thread.Sleep(8000);
+                Thread.Sleep(4000);
```
Comment (RaymondK-AP) [inline SPIClient/Connection.cs]: Any reason this is changed to 4000?

Comment (metinavci) [reply SPIClient/Connection.cs]: We reduced it in IOS and Java Libraries.

File: SPIClient/PayAtTable.cs [hunk 2]
```diff
@@ -262,18 +262,29 @@ public class GetOpenTablesResponse
...
+            }
+
+            var bphStr = JsonConvert.SerializeObject(ph);
+            return Convert.ToBase64String(Encoding.UTF8.GetBytes(bphStr));
+        }
+
         internal List<OpenTablesEntry> GetOpenTables()
         {
-            if (string.IsNullOrWhiteSpace(TableData))
+            if (OpenTablesEntries.Count == 0)
```
Comment (RaymondK-AP) [inline SPIClient/PayAtTable.cs]: Do you need to check for null here?

Comment (metinavci) [reply SPIClient/PayAtTable.cs]: Checking the count of the List is better. Because it is initiated already.

**Pull request merged**